fuuk/people/migrations/0003_refactor_grants.py

```diff
@@ -0,0 +1,156 @@
+                ('grant', models.ForeignKey(verbose_name='grant', related_name='collaborators', to='people.Grant')),
+                ('human', models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, blank=True,
+                                            verbose_name='human', to='people.Human', null=True)),
+                ('institution', models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, blank=True,
+                                                  verbose_name='institution', to='people.Institution', null=True)),
+            ],
+            options={
+                'abstract': False,
+                'verbose_name': 'co-applicant',
+                'verbose_name_plural': 'co-applicants',
+            },
+        ),
+        migrations.AlterUniqueTogether(
+            name='grantcollaborator',
+            unique_together=set([('human', 'grant')]),
+        ),
+        migrations.AlterModelOptions(
+            name='grant',
+            options={'verbose_name': 'grant', 'verbose_name_plural': 'grants'},
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='editor',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, verbose_name='editor', blank=True,
+                                    to=settings.AUTH_USER_MODEL, null=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_prefix',
+            field=models.CharField(max_length=20, null=True, verbose_name='prefix', blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_first_name',
+            field=models.CharField(max_length=50, null=True, verbose_name='first_name', blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_last_name',
+            field=models.CharField(max_length=50, null=True, verbose_name='last_name', blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_suffix',
+            field=models.CharField(max_length=20, null=True, verbose_name='suffix', blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_institution',
+            field=models.ForeignKey(verbose_name='institution', blank=True, to='people.Institution', null=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_human',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, verbose_name='human', blank=True,
+                                    to='people.Human', null=True),
+        ),
+        migrations.AlterField(
+            model_name='grant',
+            name='number',
+            field=models.CharField(max_length=20, verbose_name='grant number'),
+        ),
```

Comment:
If only the `verbose_name` changed. It's better to change it directly in initial migration. It has no effect on database structure and is more legible.